ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -285,17 +284,21 @@ def test_source_ipahealthcheck_dogtag_ca_dogtagcertsconfigcheck(self):
         assert returncode == 0
         for check in data:
             assert check["result"] == "SUCCESS"
-            assert check["kw"]["configfile"] == TOMCAT_CFG
+            assert check["kw"]["configfile"] == paths.CA_CS_CFG_PATH
             assert check["kw"]["key"] in DEFAULT_PKI_CA_CERTS
-        self.master.run_command(["mv", TOMCAT_CFG, TOMCAT_CFG + ".old"])
+        self.master.run_command(
+            ["mv", paths.CA_CS_CFG_PATH, paths.CA_CS_CFG_PATH + ".old"]
+        )
         returncode, data = run_healthcheck(
             self.master,
             "ipahealthcheck.dogtag.ca",
             "DogtagCertsConfigCheck",
         )
         assert returncode == 1
         assert data[0]["result"] == "CRITICAL"
-        self.master.run_command(["mv", TOMCAT_CFG + ".old", TOMCAT_CFG])
+        self.master.run_command(
+            ["mv", paths.CA_CS_CFG_PATH + ".old", paths.CA_CS_CFG_PATH]
+        )
         self.master.run_command(["ipactl", "restart"])
```

Comment:
Use a constant is probably a good idea but should be done in a separate patch IMHO.

Reply:
@rcritten ok will remove this and add in a seperate patch.